instrumentation-core/servlet/src/main/java/io/opentelemetry/instrumentation/servlet/ServletHttpServerTracer.java

```diff
@@ -109,4 +109,8 @@ public void setPrincipal(Span span, HttpServletRequest request) {
       span.setAttribute(MoreAttributes.USER_NAME, principal.getName());
     }
   }
+
+  public void setContentLength(Span span, int length) {
```

Comment:
The design principal of Tracers (as opposed to old Decorators) is to expose as small API as possible. For this reason I don't really like this new public method here. I don't have better idea though right now.

Reply:
It could go in the servlet 3 advice class. I originally put it here because that seemed to be the pattern.

Reply:
I opened #733 so we can leave this as-is in this PR